vm-docker/Dockerfile

```diff
@@ -0,0 +1,117 @@
+FROM 'google/cloud-sdk:slim'
+
+# install packages 
+RUN apt-get update && apt-get install -y \
+    apt-utils \
+    default-jdk \
+    g++ \
+    less \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    libncurses5-dev \
+    pkg-config \
+    python3-venv \
+    unzip \
+    wget \
+    zlib1g-dev \
+    && \
+	# clean up apt cache
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir home/tools 
```

Comment:
use `WORKDIR /home/tools` ( https://docs.docker.com/engine/reference/builder/#workdir )
to avoid `cd home/tools` in every RUN

Reply:
Great tip!